hooks/notification/hook/model/Finding.ts

```diff
@@ -4,6 +4,7 @@
 
 export interface Finding {
   name: string;
+  description: string;
```

Comment:
Did you double check if this change is compatible with the other notification hooks? Since the tests are successfull I assume it is, however.